examples/graphql-dataloader/graphql_dataloader.server.out

```diff
@@ -0,0 +1,3 @@
+$ bal run graphql_dataloader.bal
+# Below line will get printed after executing the query operation
```

Comment:
We don't need this comment here, I guess. Shall we check the other examples and see whether they use such comments?

Reply:
+1, removed